packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -251,7 +251,7 @@ const DomainPicker: FunctionComponent< Props > = ( {
 									<SuggestionNone />
 								) ) }
 							{ ! paidSuggestions &&
-								times( quantity - 1, ( i ) => <SuggestionItemPlaceholder key={ i } /> ) }
+								times( quantity, ( i ) => <SuggestionItemPlaceholder key={ i } /> ) }
```

Comment:
This change doesn't seem to be well supported by the placeholder items causing a jump: https://cloudup.com/c9q7B6BXjrx